src/config.rs

```diff
@@ -170,6 +170,27 @@ configuration_option_enum! { WriteMode:
     Checkstyle,
 }
 
+// How to reorder use statements.
+configuration_option_enum! { ImportReordering:
+    // Don't change `use` statements at all.
+    None,
+    // Just order `use` statements by their path.
+    Lines,
+    // Just order items within `use` statements by their name.
+    Items,
+    // Order `use` statements by their path and items within `use` statements by their name.
+    LinesAndItems
+}
```

Comment:
I would prefer to add another option, rather than have a single enum like this - this way lies combinatorial explosion!